R/repository.R

```diff
@@ -79,17 +79,32 @@
 as.data.frame.git_repository <- function(x, ...) {
     do.call("rbind", lapply(commits(x), as.data.frame))
 }
+##' @export
+as.data.frame.data_repository <- function(x, ...) {
+    NextMethod()
+}
 
 ##' Open a repository
 ##'
 ##' @param path A path to an existing local git repository.
 ##' @param discover Discover repository from path. Default is TRUE.
-##' @return A \code{git_repository} object with entries:
+##' @param project The name of of project. Refers to a local path in case of a
```

Comment:
In fact the function does not verify whether the subdirectory exists; no errors come up when it doesn't. I think a verification is needed.